Tests/Unit/Controller/RedirectRouteImportControllerTest.php

```diff
@@ -40,14 +40,13 @@ class RedirectRouteImportControllerTest extends TestCase
 
     public function testImportAction()
     {
-        $importFile = $this->importPath . '/' . $this->fileName;
-
         $request = $this->prophesize(Request::class);
 
         $fileBag = $this->prophesize(FileBag::class);
         $request->reveal()->files = $fileBag->reveal();
 
         $file = new File($fileName = __DIR__ . '/import.csv');
+        $importFile = $file->getPathname();
```

Comment:
https://travis-ci.org/github/sulu/SuluRedirectBundle/jobs/701256884 This error happened without this change